### DISCUSSION TITLE
アクリル板について

### DISCUSSION THREAD

**Discussion by tarboh** [Yowkees/keyball#451] [Q&A] [answered]
先日Keyball61を購入させていただきました。

届いたキットのなかで、トラックボールが付かない方のトッププレートとミドルプレートの間のアクリル板に傷のようなものが入っていました。これは商品仕様の範囲内になりますでしょうか？

実用に支障がある訳では無いのですが、念のため報告を兼ねてご連絡させていただきました。
![プレート画像３](https://github.com/Yowkees/keyball/assets/50641757/84066be7-a4ab-4e64-b29e-3eb4b9682de4)
![プレート画像２](https://github.com/Yowkees/keyball/assets/50641757/0938e849-f587-4693-a04a-0c4e461cc109)
![プレート画像１](https://github.com/Yowkees/keyball/assets/50641757/3a337eab-1098-43da-9dae-000058fda4a9)

**Comment by Ryo624** [comment, marked as answer]
ご連絡ありがとうございます。

アクリル板に傷が入っている物は初期不良のものと思われます。
ご迷惑をおかけしまして大変申し訳ありません。

新しいアクリルプレートとの取り換えが可能ですので、tarboh様がご希望の場合、白銀ラボのウェブショップにご連絡いただけますでしょうか。

**Reply by tarboh** [reply]
早速のご回答ありがとうございます。

折角ですので、取り換えをお願いしたいと思います。
ウェブショップへご連絡させていただきますので、どうぞよろしくお願いいたします。
